src/Nexus.ts

```diff
@@ -649,10 +693,35 @@ class Nexus {
           return this.request(url, args, method);
         }
       }
+      if (err instanceof JwtExpiredError && this.jwtRefreshTries < param.MAX_JWT_REFRESH_TRIES) {
+        this.jwtRefreshTries++;
+        return this.handleJwtRefresh().then((newOAuthCredentials) => {
```

Comment:
In this library we use async/await for asynchronous calls instead of Promise.then so this should be
```
const newOAuthCredentials = await this.handleJwtRefresh();
...
```